server/persistence/interface.go

```diff
@@ -49,11 +52,78 @@ func (d *db) AddPlayerColorToGame(pID model.PlayerID, color model.PlayerColor, g
 }
 
 func (d *db) GetGame(id model.GameID) (model.Game, error) {
-	return d.games.Get(id)
+	g, err := d.games.Get(id)
+	if err != nil {
+		return model.Game{}, err
+	}
+
+	err = d.overwritePlayers(g)
+	if err != nil {
+		return model.Game{}, err
+	}
+
+	return g, nil
+}
+
+func (d *db) overwritePlayers(g model.Game) error {
+	for i, player := range g.Players {
+		// overwrite the player that the game service knows
+		// about with the player that the players service knows about
```

Comment:
Why do we have to overwrite the players? Does the game service only get the IDs and not the full player?

Reply:
The game service should not be in charge of the players. The players service should be the authority on players. The game should just know what players are in it.